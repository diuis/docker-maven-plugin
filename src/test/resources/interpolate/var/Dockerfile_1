FROM ${base}
LABEL name ${name} @age@ ${ext} ${unknown}
ENV ${name} ${age}
