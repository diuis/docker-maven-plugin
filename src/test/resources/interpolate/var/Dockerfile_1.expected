FROM java
LABEL name guenther @age@ png ${unknown}
ENV guenther 42
